doc/operations/troubleshooting.rst

```diff
@@ -88,4 +88,14 @@ There is an option to generate machine-readable logs for monitoring. To enable t
 - ``level`` is the level of the log message [info, debug, trace, fail, fatal]
 - ``msg`` is the log message
 
-See :ref:`this page <build_apps/logging:Logging>` for steps to add application-specific logging, which will have an additional ``tag`` field set to ``app``.
\ No newline at end of file
+See :ref:`this page <build_apps/logging:Logging>` for steps to add application-specific logging, which will have an additional ``tag`` field set to ``app``.
+
+Error Codes
+-----------
+
+StartupSeqnoIsOld
+~~~~~~~~~~~~~~~~~
+
+Returned when a node tries to join a network with too old a snapshot. See :ref:`this page <operations/ledger_snapshot:Join or Recover From Snapshot>` for more information.
```

Comment:
or no snapshot? We could maybe reference https://microsoft.github.io/CCF/main/operations/operator_rpc_api.html#get--node-state (`"startup_seqno"` entry).